### PR TITLE
Adding quick start guide

### PR DESCRIPTION
**Pull request by erundle**
Adding quick start guide

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Within the download you'll find the following directories and files, logically grouping common assets and providing both compiled and minified variations. You'll see something like this:
+
+```
+patternfly/
+├── components/
+│   ├── 3rd Party Repos. May need to pull in additional JS includes based on Patternfly Usage (No additional CSS includes needed)
+├── dist/
+│   ├── css/
+│   │   │── patternfly.min.css (* Need to include)
+│   │   │── patternnfly-additions.min.css (* Need to include)
```
Comment (rhamilto) [inline QUICKSTART.md]: "patternnfly" has an extra "n"

File: QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    ├── Example Code Src Files
+```
+
+We provide compiled CSS and JS (`patternfly.*`), as well as compiled and minified CSS and JS (`pattternfly.min.*`). CSS [source maps](https://developer.chrome.com/devtools/docs/css-preprocessors) (`patternfly.*.map`) are available for use with certain browsers' developer tools. Fonts from OpenSans, FontAwesome, Patternfly Icons are included, as well as numerous other 3rd party JS components and libraries.
+
+## Using Patternfly In Your Application
+
+1. Ensure you have installed the following utilities:
+    - [Git](http://git-scm.com/downloads): a free and open source distributed version control system.
+        - Unfamiliar with Git? I recommend using [GitHub for Windows](https://windows.github.com/) or [GitHub for Mac](https://mac.github.com/).
```
Comment (rhamilto) [inline QUICKSTART.md]: It's not clear who "I" is here.  Maybe "Try using" instead of "I recommend"?

File: QUICKSTART.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+4. Add the following script includes to your HTML file(s), adjusting where necessary to pull in only what you need:
+
+        <!-- jQuery -->
+        <script src="bower_components/patternfly/components/jquery/dist/jquery.js"></script>
+
+        <!-- Bootstrap JS -->
+        <script src="bower_components/patternfly/components/bootstrap/dist/js/bootstrap.js"></script>
+
+        <!-- Patternfly Custom Componets -  Sidebar, Popovers and Datatables Customizations -->
+        <script src="bower_components/patternfly/dist/js/patternfly.js"></script>
```
Comment (rhamilto) [inline QUICKSTART.md]: This script needs to be called last as it contains datatables config stuff.

**Comment by dtaylor113** [comment]
LGTM!

**Pull request merged**